core/src/test/scala/integration/kafka/api/PlaintextAdminIntegrationTest.scala

```diff
@@ -2714,6 +2714,56 @@ class PlaintextAdminIntegrationTest extends BaseAdminIntegrationTest {
       ConfigSource.DYNAMIC_TOPIC_CONFIG, false, false, Collections.emptyList(), null, null),
       topicConfigs.get(TopicConfig.INDEX_INTERVAL_BYTES_CONFIG))
   }
+
+  @ParameterizedTest
+  @ValueSource(strings = Array("kraft"))
+  def testAlterReassignmentsWithBootstrapControllers(quorum: String): Unit = {
+    client = Admin.create(createConfig)
+    val assignments = new util.HashMap[Integer, util.List[Integer]]
+    assignments.put(0, util.Arrays.asList(0, 1, 2))
+    assignments.put(1, util.Arrays.asList(1, 2, 0))
+    assignments.put(2, util.Arrays.asList(2, 1, 0))
+    val createTopicResult = client.createTopics(Collections.singletonList(new NewTopic("foo", assignments)))
```

Comment:
nit: Maybe we could consider extracting foo as a variable? It could improve readability and prevent accidental changes.

Reply:
Sure, I'will address this after moving the the whole test cases to `BootstrapControllersIntegrationTest`, thanks